scrapy/link.py

```diff
@@ -7,7 +7,21 @@
 
 
 class Link:
-    """Link objects represent an extracted link by the LinkExtractor."""
+    """Link objects represent an extracted link by the LinkExtractor.
+
+    Using the anchor tag sample below to illustrate the parameters::
+
+            <a href="/nofollow.html#foo" rel="nofollow">Dont follow this one</a>
+
+    :param url: the address being linked to in the anchor tag. From the sample, this is ``base_url/nofollow.html``.
```

Comment:
If `url` is meant to be an absolute URL, as it seems, I would make that explicit here (“address” → “absolute URL”).

As for `base_url`, I would turn it into `<base_url>` to make it a bit more obvious that it is a place holder. Alternatively, you may even consider going further, and suggest that, assuming the HTML document containing the anchor was https:/example.com/index.html, the URL would be ``https://example.com/nofollow.html``.